package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@eppo/js-client-sdk",
-  "version": "3.8.3",
+  "version": "3.9.0-alpha.0",
```

Comment:
⚠️ Do not publish this version for dogfooding until 

1. The edge compute endpoint is published with updated hashing
2. The corresponding md5 hasher in js-client-sdk-common is updated https://github.com/Eppo-exp/js-sdk-common/pull/164/files#r1887291803 (`.appendBinary` --> `.append`)
3. And 4.7.0-alpha.1 version of js-client-sdk-common has been released and the dependency in this file has been updated to match